database/schema.sql

```diff
@@ -2,6 +2,7 @@ CREATE TABLE certificates(
 	id	        				serial primary key,
 	sha1_fingerprint	        varchar NOT NULL,
 	sha256_fingerprint	    	varchar NOT NULL,
+    sha256_subject_spki         varchar NOT NULL,
```

Comment:
Mixing tabs and spaces is never a good idea. I moved the entire file to spaces only.